themes/tailwind-audio/template.html

```diff
@@ -0,0 +1,227 @@
+      2.561-1.06V4.06zM18.584 5.106a.75.75 0 011.06 0c3.808 3.807 3.808 9.98 0 13.788a.75.75 0 11-1.06-1.06
+      8.25 8.25 0 000-11.668.75.75 0 010-1.06z"
+    ></path>
+    <path
+      d="M15.932 7.757a.75.75 0 011.061 0 6 6 0 010 8.486.75.75 0 01-1.06-1.061 4.5 4.5 0 000-6.364.75.75 0
+      010-1.06z"
+    ></path>
+  </symbol>
+
+  <symbol id="off" viewBox="0 0 24 24">
+    <path
+      d="M13.5 4.06c0-1.336-1.616-2.005-2.56-1.06l-4.5 4.5H4.508c-1.141 0-2.318.664-2.66 1.905A9.76 9.76 0
+      001.5 12c0 .898.121 1.768.35 2.595.341 1.24 1.518 1.905 2.659 1.905h1.93l4.5 4.5c.945.945 2.561.276
+      2.561-1.06V4.06zM17.78 9.22a.75.75 0 10-1.06 1.06L18.44 12l-1.72 1.72a.75.75 0 001.06 1.06l1.72-1.72 1.72
+      1.72a.75.75 0 101.06-1.06L20.56 12l1.72-1.72a.75.75 0 00-1.06-1.06l-1.72 1.72-1.72-1.72z"
+    />
+  </symbol>
+</svg>
+
+<media-controller
+  audio
+  defaultsubtitles="{{defaultsubtitles}}"
+  defaultduration="{{defaultduration}}"
+  gesturesdisabled="{{disabled}}"
+  hotkeys="{{hotkeys}}"
+  nohotkeys="{{nohotkeys}}"
+  defaultstreamtype="on-demand"
+  class="@container block w-full shadow-xl shadow-black/5"
+  style="
+    --media-background-color: transparent;
+    --media-control-background: transparent;
+    --media-control-hover-background: transparent;
+  "
+>
+  <slot name="media" slot="media"></slot>
+
+  <media-time-range
+    class="block @md:hidden w-full h-2 min-h-0 p-0 bg-slate-50 focus-visible:ring-slate-700 focus-visible:ring-2"
+    style="
+      --media-range-track-background: transparent;
+      --media-time-range-buffered-color: rgb(0 0 0 / 0.02);
+      --media-range-bar-color: rgb(79 70 229);
+      --media-range-track-height: 0.5rem;
+      --media-range-thumb-background: rgb(79 70 229);
+      --media-range-thumb-box-shadow: 0 0 0 2px rgb(255 255 255 / 0.9);
+      --media-range-thumb-width: 0.25rem;
+      --media-range-thumb-height: 1rem;
+      --media-preview-time-text-shadow: transparent;
+    "
+  >
+    <media-preview-time-display
+      slot="preview"
+      class="text-slate-600 text-xs"
+    ></media-preview-time-display>
+  </media-time-range>
+
+  <media-control-bar
+    class="h-20 @md:h-16 px-4 w-full bg-white flex items-center justify-between @md:rounded-md @md:ring-1 @md:ring-slate-700/10"
+  >
+    <media-seek-backward-button
+      seekoffset="10"
+      class="w-8 h-8 p-0 group rounded-full focus:outline-none focus-visible:ring-slate-700 focus-visible:ring-2"
+    >
+      <svg
+        slot="icon"
+        aria-hidden="true"
+        class="w-7 h-7 fill-none stroke-slate-500 group-hover:stroke-slate-700"
+      >
+        <use href="#backward" />
+      </svg>
+    </media-seek-backward-button>
+
+    <media-play-button
+      class="h-10 w-10 p-2 mx-3 rounded-full bg-slate-700 hover:bg-slate-900 focus:outline-none focus:ring-slate-700 focus:ring-2 focus:ring-offset-2"
+    >
+      <svg slot="play" aria-hidden="true" class="relative left-px">
+        <use href="#play" />
```

Comment:
This is interesting. Is there a benefit to this sprite approach for SVGs vs. just having the SVG path here? I guess if you're using it twice in the theme?

Reply:
indeed, benefit is for re-use and maybe more readable HTML.